build-system/tasks/coverage-map/index.js

```diff
@@ -116,14 +139,17 @@ coverageMap.description =
 coverageMap.flags = {
   json:
     '  Customize the name of the JSON output from puppeteer (out.json by default).',
-  url:
-    '  Set the URL for puppeteer testing, starting with  "http://localhost[:port_number]..." (http://localhost[:port_number]/examples/everything.amp.html by default).',
-  html:
+  inputhtml:
+    '  Set the input HTML for puppeteer testing, by designating the path that leads to the HTML file, starting at "examples/" (everything.amp.html by default).',
+  outputhtml:
     '  Customize the name of the HTML output from source map explorer (out.html by default).',
-  nobuild:
-    "  Skips dist build. Your working directory should be dist if you're using this flag.",
+  nobuild: '  Skips dist build.',
   port:
     '  Customize the port number of the local AMP server (8000 by default).',
   file:
-    '  Designate which JS file to view in coverage map, or *.js for all files (v0.js by default). If the JS file is not in the top level dist directory, you need to indicate the path to the JS file relative to dist.',
+    '  Designate which JS (or MJS) file to view in coverage map, or *.js for all files (v0.js by default). If the JS file is not in the top level dist directory, you need to indicate the path to the JS file relative to dist.',
+  esm:
+    '  Perform coverage test in ESM environment. This will trigger an additional HTML transformation.',
```

Comment:
What should happen if I provide `--file *.mjs` without `--esm` (or the reverse `--file *.js --esm`)?

Reply:
This would not work, because the HTML would not point to the correct file.
There is currently a bug in the transformer that does not correctly transform the HTML.
What I can do is to change the input file based on the presence of the `esm` flag.